scss/2-tools/_typography.scss

```diff
@@ -4,35 +4,15 @@
     update the list of elements that we want to check against.
 */
 @mixin cads-adjacent-heading-margin($m) {
```

Comment:
Should this set a default spacing? Seems to be `$cads-spacing-6`?

Reply:
I can do - wonder if forcing a consumer to specify is better in the long run though? 🤷

Reply:
I guess it comes down to if the adjacent margin rules are a fixed design, or something that should be flexible. Reckon it's fine go with this for now though